sky/provision/lambda_cloud/lambda_utils.py

```diff
@@ -163,27 +166,27 @@ def create_instances(self,
             'instance_type_name': instance_type,
             'ssh_key_names': [ssh_key_name],
             'quantity': quantity,
-            'name': name
+            'name': name,
         })
         response = _try_request_with_backoff(
             'post',
             f'{API_ENDPOINT}/instance-operations/launch',
             data=data,
-            headers=self.headers)
+            headers=self.headers,
+        )
         return response.json().get('data', []).get('instance_ids', [])
 
     def remove_instances(self, *instance_ids: str) -> Dict[str, Any]:
         """Terminate instances."""
-        data = json.dumps({
-            'instance_ids': [
-                instance_ids[0]  # TODO(ewzeng) don't hardcode
-            ]
-        })
+        data = json.dumps({'instance_ids': [instance_ids[0]]
+                          }  # TODO(ewzeng) don't hardcode
+                         )
```

Comment:
```suggestion
        # TODO(ewzeng) don't hardcode
        data = json.dumps({'instance_ids': [instance_ids[0]]})
```
nit

Reply:
good catch here esp since we're always passing down a list now so need to grab all instance ids